src/main/target/SITL/target.h

```diff
@@ -115,17 +115,12 @@
 
 #define FLASH_SIZE 2048
 
-# define DEFIO_PORT_USED_COUNT 0
-# define DEFIO_PORT_USED_LIST /* empty */
-# define DEFIO_PORT_OFFSET_LIST /* empty */
 
 #define LED_STRIP_TIMER 1
 #define SOFTSERIAL_1_TIMER 2
 #define SOFTSERIAL_2_TIMER 3
 
-#define TARGET_IO_PORTA         0xffff
-#define TARGET_IO_PORTB         0xffff
-#define TARGET_IO_PORTC         0xffff
+#define TARGET_IO_PORTA         0x0001   // define one pin to avoid warnings
```

Comment:
It's a shame that we have to define fake hardware just to satisfy some standard. Isn't there a way to do this that is more intuitive?